contracts/upgradeable_contracts/BasicHomeBridge.sol

```diff
@@ -22,7 +22,8 @@ contract BasicHomeBridge is EternalStorage, Validatable, BasicBridge, BasicToken
         uint256 NumberOfCollectedSignatures
     );
 
-    function executeAffirmation(address recipient, uint256 value, bytes32 transactionHash) external onlyValidator {
+    function executeAffirmation(address recipient, uint256 value, bytes32 transactionHash) public onlyValidator {
```

Comment:
Is there any reason to update the modifier of `executeAffirmation` from `external` to `public`?